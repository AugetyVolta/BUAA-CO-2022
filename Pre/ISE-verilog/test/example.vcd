$date
	Thu Aug 25 21:14:12 2022
$end
$version
	Icarus Verilog
$end
$timescale
	1ps
$end
$scope module tb_chardet $end
$var reg 1 ! a $end
$var reg 1 " b $end
$var reg 1 # c $end
$scope module u_chardet $end
$var wire 1 ! a $end
$var wire 1 " b $end
$var wire 1 # c $end
$upscope $end
$upscope $end
$enddefinitions $end
#0
$dumpvars
0#
0"
0!
$end
#10000
1#
1!
#20000
0#
1"
0!
#30000
1!
