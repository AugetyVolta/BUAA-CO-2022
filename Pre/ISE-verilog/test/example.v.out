#! c:/iverilog-x86/bin/vvp
:ivl_version "10.0 (stable)" "(v10_0)";
:ivl_delay_selection "TYPICAL";
:vpi_time_precision + 0;
:vpi_module "system";
:vpi_module "vhdl_sys";
:vpi_module "v2005_math";
:vpi_module "va_math";
S_001c39e8 .scope module, "chardet" "chardet" 2 1;
 .timescale 0 0;
    .port_info 0 /INPUT 1 "a"
    .port_info 1 /INPUT 1 "b"
    .port_info 2 /INPUT 1 "c"
o02911874 .functor BUFZ 1, C4<z>; HiZ drive
v001c3ae8_0 .net "a", 0 0, o02911874;  0 drivers
o0291188c .functor BUFZ 1, C4<z>; HiZ drive
v001c5580_0 .net "b", 0 0, o0291188c;  0 drivers
o029118a4 .functor BUFZ 1, C4<z>; HiZ drive
v001c55d8_0 .net "c", 0 0, o029118a4;  0 drivers
# The file index is used to find the file name in the following table.
:file_names 3;
    "N/A";
    "<interactive>";
    "example.v";
